de.atb.typhondl.acceleo/src/de/atb/typhondl/acceleo/files/generateDockerComposeDeployment.mtl

```diff
@@ -114,6 +122,47 @@ volumes:
   [aContainer.printMongoRep()/][aContainer.printMongoInitFiles()/][/if]
 [/template]
 
+[template public fluentdImage(aContainer : Container)]
+# fluentd/Dockerfile
+
+FROM fluent/fluentd:v1.9-debian-1
+USER root
+RUN ['['/]"gem", "install", "fluent-plugin-elasticsearch", "--no-document", "--version", "4.2.0"[']'/]
+# COPY fluent.conf /fluentd/etc/fluent.conf
+USER fluent
+[/template]
+
+[template public fluentConf(aContainer : Container)]
+# fluent.conf
+
+<source>
+  @type forward
+  bind 0.0.0.0
+  port 24224
+</source>
+
+<match *.**>
+  @type copy
+
+  <store>
+    @type elasticsearch
+    host elasticsearch
```

Comment:
Can we customize this field and allow the user to type in a custom host if they are not interested in deploying elasticsearch within the application? If you let me know where I can try to create a pull request on top of this one to add the functionality

Reply:
Yes we can, good idea :+1: 
I will add an additional field to the wizard